src/components/PostDetails.tsx

```diff
@@ -98,21 +62,21 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
       </div>
 
       <div className="block">
-        {!loaded && <Loader />}
+        {!loading && <Loader />}
 
-        {loaded && hasError && (
+        {loading && err && (
           <div className="notification is-danger" data-cy="CommentsError">
             Something went wrong
           </div>
         )}
```

Comment:
The condition for displaying the error message is incorrect. It should be shown when `loading` is false and `err` is true. Consider changing the condition to `{!loading && err && ( ... )}`.